docs/src/docs/snippets/gradle/groovy/build.gradle

```diff
@@ -99,7 +99,7 @@ graalvmNative {
         // Copies metadata collected from tasks into the specified directories.
         metadataCopy {
             inputTaskNames.add("test") // Tasks previously executed with the agent attached.
-            outputDirectories.add("src/main/resources/META-INF/native-image")
+            outputDirectories.add("src/main/resources/META-INF/native-image/<groupId>/<artifactId>/") // Replace <groupId> and <artifactId> with GAV coordinates of your project
```

Comment:
Is the trailing forward slash correct? We used to not have it.

Reply:
Since `src/main/resources/META-INF/native-image/<groupId>/<artifactId>/` is directory, I don't see any difference if we use the trailing forward slash or not. It should work in any case, and I think it doesn't change anything. We can remove it if it is necessary